### ISSUE TITLE
Add possibility to place card controls (links & buttons) in a card footer

### ISSUE THREAD

**Issue by DC-RomanKarpov** [gravity-ui/page-constructor#826]
This feature is required by our content managers to align all control in cards, for example, see the attached screenshots.

<img width="979" alt="Screenshot 2024-02-22 at 14 54 48" src="https://github.com/gravity-ui/page-constructor/assets/124568536/8fef646c-9016-4b15-bfed-8250a8605c65">

<img width="1514" alt="Screenshot 2024-02-22 at 14 54 36" src="https://github.com/gravity-ui/page-constructor/assets/124568536/5c34ffdc-1f6f-46ac-bde9-4ea45b62f54c">

For impl example, please, see 
https://preview.gravity-ui.com/page-constructor/823/?path=/story/components-cards-backgroundcard--control-position
https://preview.gravity-ui.com/page-constructor/823/?path=/story/components-cards-basiccard--control-position
https://preview.gravity-ui.com/page-constructor/823/?path=/story/components-cards-layoutitem--control-position

**Comment by stepastepastepa** [comment]
@DC-RomanKarpov does this only work for specific cards? We need to remove this property from the background card and add it to the image card. Have we touched the original content component? I see that the layout of the buttons has changed at least.

**Comment by stepastepastepa** [comment]
Yes, you're right. There are no buttons. Then we don't touch that place.

To the account of the **background-card**. Is it possible to use the new parameter only if we do not specify **paddingBottom**? When this parameter is present, the alignment looks very strange. It is necessary to make such a combination impossible to assemble. Could we do that?

There are no visual changes. I noticed because now the indentation is implemented differently. I just need to understand exactly what we have touched on in order to check other cases. Everything is ok now.
